app/org/sagebionetworks/bridge/BridgeUtils.java

```diff
@@ -397,5 +397,11 @@ public static String createReferentGuidIndex(Activity activity, LocalDateTime lo
         }
         throw new BridgeServiceException("Invalid activityType specified");    
     }
-
+    
+    public static String toSynapseFriendlyName(String input) {
+        if (input == null) {
+            return "";
+        }
```

Comment:
I'm pretty sure "" is probably not a Synapse-friendly name either. (Or if it is, it's not a particularly useful one.) Would it be better to just throw on a null input?

Reply:
This now throws exceptions now if the value is blank or null. It could happen if the study name was particularly strange, but almost certainly will not. Also added tests to verify these are converted to BadRequestException by the service.